1.data_preprocessing/2.format_bulk.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    import subprocess
</pre><p>where are you using subprocess? just curious</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NDavidsonLab/BuDDI_HGSC_schildkraut_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>